packages/salesforcedx-vscode-core/src/sfdxProject/sfdxPackageDirectories.ts

```diff
@@ -27,7 +27,11 @@ export default class SfdxPackageDirectories {
             if (dirPath.startsWith(path.sep)) {
               dirPath = dirPath.substring(1);
             }
-            packageDirectoryPaths.push(dirPath);
+            if (packageDirectory.default) {
+              packageDirectoryPaths = [dirPath].concat(packageDirectoryPaths);
```

Comment:
Slight modification to have the default package directory in a project be the first element of the list.